mmf/configs/zoo/models.yaml

```diff
@@ -8,6 +8,20 @@ visual_bert:
         hashcode: 9af0b8101579d7587d70d7315940310c7fc5ef7269cba497780922e65e4e000d
     defaults: ${visual_bert.pretrained.coco}
 
+  finetuned:
+    hateful_memes_direct:
+      version: 1.0_2020_05_10
+      resources:
+      - url: mmf://models/visual_bert/visual_bert.finetuned.hateful_memes_direct.tar.gz
+        file_name: visual_bert.finetuned.hateful_memes_direct.tar.gz
+        hashcode: 1780582b65c0e5ab383bbf862f140daaa828963c38710f9cc004a7d883b219f8
+    hateful_memes_from_coco:
```

Comment:
I would also add `defaults` to all of these. So that once person says `vilbert.hateful_memes` they get best variation directly. Also, they would not need to mention full key then. You can use interpolation for this by directly setting defaults to best node.